blockchain/block/blockstore.go

```diff
@@ -7,6 +7,8 @@
 package block
 
 import (
+	"github.com/iotexproject/iotex-core/pkg/log"
```

Comment:
move this line together to after line 16, group internal packages together

Reply:
done